_data/accessibility-tests/header.yml

```diff
@@ -0,0 +1,243 @@
+title: Header
+component_status: pass
+test_items:
+# General tests
+  - summary:
+    summary_additional:
+    test_status:
+    test_type: general
+    version_tested:
+    wcag_criterion:
+  - summary: Header content is consistent in mobile views.
+    summary_additional: |
+      When you view the page on a mobile device, 
+      the header is visible and works in both portrait and landscape settings.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.4
+  - summary: Text in header meets color contrast requirements.
+    summary_additional: |
+      When you use a color contrast checker to view the header,
+      contrast between the text color and background color is at least 4.5:1.
+    test_status: 3.8.1
+    test_type: general
+    version_tested: pass
+    wcag_criterion: 1.4.3
+  - summary: Header remains visible and functional when screen is magnified. 
+    summary_additional: |
+      When you zoom in to 400%, 
+      you can still see and use the header menu and navigation, 
+      and it doesn't overlap with or get cut off by any other content.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Search icon in header meets color contrast requirements.
+    summary_additional: |
+      When you use a color contrast checker to view the search icon, 
+      contrast between the icon and adjacent colors is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
+  - summary: Current page link has a distinct style.
+    summary_additional: |
+      When viewing the navigation link list, 
+      the current page has distinct styling that gives clear visual indication of where you are in the site.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.8
```

Comment:
@RachelCorsino 
**issue:** This criterion needs to be added to [wcag-details.html](https://github.com/uswds/uswds-site/blob/main/_includes/accessibility-tests/wcag-details.html). Can you make this update?

Reply:
Resolved in [6eece60](https://github.com/uswds/uswds-site/pull/2761/commits/6eece608e8f483998c24878cc20e38ecab66eca1)